### PR TITLE
Add link to issue #5756

### PR DESCRIPTION
**Pull request by detherminal**
Added the link to issue #5756

### CODE REVIEW

**Review by lildude — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ bundle exec bin/github-linguist --breakdown
 
 We try only to add new extensions once they have some usage on GitHub.
 In most cases we prefer that each new file extension be in use in at least 200 unique `:user/:repo` repositories before supporting them in Linguist
-(but see #5756 for a temporary change in the criteria).
+(but see [#5756](https://github.com/github/linguist/issues/5756) for a temporary change in the criteria).
```
Comment (Alhadis) [inline CONTRIBUTING.md:67]: This link's been used twice, so I suggest keeping it DRY:

```suggestion
(but see [#5756][] for a temporary change in the criteria).
```

~~~diff 
 [grammars]: /vendor/README.md
 [heuristic]: https://github.com/github/linguist/blob/master/lib/linguist/heuristics.yml
 [languages]: /lib/linguist/languages.yml
 [licenses]: https://github.com/github/linguist/blob/9b1023ed5d308cb3363a882531dea1e272b59977/vendor/licenses/config.yml#L4-L15
 [new-issue]: https://github.com/github/linguist/issues/new
 [samples]: /samples
 [search-example]: https://github.com/search?utf8=%E2%9C%93&q=extension%3Aboot+NOT+nothack&type=Code&ref=searchresults
 [gpr]: https://docs.github.com/packages/using-github-packages-with-your-projects-ecosystem/configuring-rubygems-for-use-with-github-packages
+[#5756]: https://github.com/github/linguist/issues/5756
~~~

**Review by Alhadis — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ Additionally, if this extension is already listed in [`languages.yml`][languages
 
 We try only to add languages once they have some usage on GitHub.
 In most cases we prefer that each new file extension be in use in at least 200 unique `:user/:repo` repositories before supporting them in Linguist
-(but see #5756 for a temporary change in the criteria).
+(but see [#5756](https://github.com/github/linguist/issues/5756) for a temporary change in the criteria).
```
Comment (Alhadis) [inline CONTRIBUTING.md:93]: As above:

```suggestion
(but see [#5756][] for a temporary change in the criteria).
```

**Review by Alhadis — CHANGES_REQUESTED** [review]

**Comment by detherminal** [comment]
@Alhadis Added it

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -64,7 +64,7 @@ bundle exec bin/github-linguist --breakdown
 
 We try only to add new extensions once they have some usage on GitHub.
 In most cases we prefer that each new file extension be in use in at least 200 unique `:user/:repo` repositories before supporting them in Linguist
-(but see [#5756](https://github.com/github/linguist/issues/5756) for a temporary change in the criteria).
+(but see [#5756][5756] for a temporary change in the criteria).
```
Comment (Alhadis) [inline CONTRIBUTING.md:63]:
```suggestion
(but see [#5756][] for a temporary change in the criteria).
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -90,7 +90,7 @@ Additionally, if this extension is already listed in [`languages.yml`][languages
 
 We try only to add languages once they have some usage on GitHub.
 In most cases we prefer that each new file extension be in use in at least 200 unique `:user/:repo` repositories before supporting them in Linguist
-(but see [#5756](https://github.com/github/linguist/issues/5756) for a temporary change in the criteria).
+(but see [#5756][5756] for a temporary change in the criteria).
```
Comment (Alhadis) [inline CONTRIBUTING.md:89]:
```suggestion
(but see [#5756][] for a temporary change in the criteria).
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -218,3 +218,4 @@ As Linguist is a production dependency for GitHub we have a couple of workflow r
 [samples]: /samples
 [search-example]: https://github.com/search?utf8=%E2%9C%93&q=extension%3Aboot+NOT+nothack&type=Code&ref=searchresults
 [gpr]: https://docs.github.com/packages/using-github-packages-with-your-projects-ecosystem/configuring-rubygems-for-use-with-github-packages
+[5756]: https://github.com/github/linguist/issues/5756
```
Comment (Alhadis) [inline CONTRIBUTING.md:217]:
```suggestion
[#5756]: https://github.com/github/linguist/issues/5756
```

**Review by Alhadis — CHANGES_REQUESTED** [review]

**Review by Alhadis — APPROVED** [review]

**Pull request merged**